packages/flutter/lib/src/material/scaffold.dart

```diff
@@ -1228,8 +1228,8 @@ class ScaffoldState extends State<Scaffold> with TickerProviderStateMixin {
     }
   }
 
-  PersistentBottomSheetController<T> _buildBottomSheet<T>(WidgetBuilder builder, AnimationController controller, bool isLocalHistoryEntry) {
-    final Completer<T> completer = Completer<T>();
+  PersistentBottomSheetController<void> _buildBottomSheet(WidgetBuilder builder, AnimationController controller, bool isLocalHistoryEntry) {
+    final Completer<void> completer = Completer<void>();
```

Comment:
This completer only completes without value and it seems to be the only place where `T` was used.